src/main/webapp/js/omniComment.js

```diff
@@ -410,7 +410,7 @@ function deleteComment(commentIdx){
     }
 }
 
-function enableEdit(commentIdx){
+function enableEdit(commentIdx, unusedParameter) {
```

Comment:
I wish I don't have to add it, but the other `enableEdit` function (in `instructorStudentRecords.js`) has 2 params XP